NOTICE.md

```diff
@@ -0,0 +1,135 @@
+# Notices for Eclipse Modeling Workflow Engine
+
+This content is produced and maintained by the Eclipse Modeling Workflow Engine
+project.
+
+* Project home: https://projects.eclipse.org/projects/modeling.emf.mwe
+
+## Trademarks
+
+Eclipse Modeling Workflow Engine, and Modeling Workflow Engine are trademarks of
+the Eclipse Foundation.
+
+## Copyright
+
+All content is the property of the respective authors or their employers. For
+more information regarding authorship of content, please consult the listed
+source code repository logs.
+
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v. 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0.
+
+SPDX-License-Identifier: EPL-2.0
+
+## Source Code
+
+The project maintains the following source code repositories:
+
+* http://git.eclipse.org/c/emf/org.eclipse.mwe.git
+* https://github.com/eclipse/mwe
+
+## Third-party Content
+
+This project leverages the following third party content.
+
+ANTLR (3.0)
+
+* License: New BSD license
+
+AOP Alliance (1.0)
+
+* License: Public Domain
+
+Apache Commons CLI (1.2.0)
+
+* License: Apache License, 2.0
+* Project: http://commons.apache.org/cli/index.html
+* Source: http://svn.apache.org/viewvc/commons/proper/cli/trunk/
+
+Apache Commons Lang (2.4)
+
+* License: Apache License, 2.0 
+
+Apache Commons Lang (2.6)
+
+* License: Apache License, 2.0
+
+Apache Commons Line Interface (1.0)
+
+* License: Apache Software License 1.1 
+
+Apache Commons Logging Jar (1.1.1)
+
+* License: Apache License, 2.0 
+
+asm (6.2)
+
+* License: BSD-3-Clause
+
+ASM (7.2)
+
+* License: BSD-3-Clause
+
+asm 7 (n/a)
+
+* License: BSD-3-Clause
+
+asm 7 (n/a)
```

Comment:
duplikate?

Reply:
@miklossy please open against the tool the eclipse foundation provides

Reply:
and then send me the bugzilla number

Reply:
I'll see. Than the change is fine!